mne/viz/backends/_notebook.py

```diff
@@ -1565,8 +1582,17 @@ def show(self):
         else:
             self._display_default_tool_bar()
         # viewer
-        viewer = self.plotter.show(jupyter_backend="ipyvtklink", return_viewer=True)
-        viewer.layout.width = None  # unlock the fixed layout
+        viewer = self.plotter.show(jupyter_backend=_JUPYTER_BACKEND, return_viewer=True)
+        if _JUPYTER_BACKEND == "trame":
+            # Remove scrollbars, see https://github.com/pyvista/pyvista/pull/4847
+            # which should be okay to use with this. Can probably remove once we require
+            # PyVista 0.43
+            viewer.value = re.sub(
+                r" style=[\"'](.+)[\"']></iframe>",
+                # value taken from matplotlib's widget
+                r" style='\1; border: 1px solid rgb(221,221,221);' scrolling='no'></iframe>",  # noqa: E501
+                viewer.value,
+            )
```

Comment:
this seems....  fragile. I read the linked issue, but didn't see an upstream report in `trame` about the scrollbars. Is there something more to do here that will (eventually) let us remove this?

Reply:
It's not `trame` that adds the `iframe` that needs to be modified, it's PyVista (they create the `iframe` in the HTML ipywidget). I'll improve the comment